### DISCUSSION TITLE
Catppuccin visual mode not visible

### DISCUSSION THREAD

**Discussion by jkopano** [LazyVim/LazyVim#4612] [Q&A]
Hello,
I use catppuccin theme, i have a problem with symbol hightlighting when in visual mode, its barely visible.

In normal mode:
![2024-10-18_23-54](https://github.com/user-attachments/assets/0f7cf945-0c15-42be-a38e-3e18d618c881)
In visual mode:
![image](https://github.com/user-attachments/assets/45be3dd1-78fb-44ee-895c-9184db83fbb0)

I look for option to disable symbol highlighting when in visual mode.
Or other solutions, anything is welcome.

PS: where can i donate to the project?

**Comment by alewis001** [comment]
RIghtly or wrongly, this is how I've tweaked some of the colors. I've added a line for Visual select to answer your question...
 
In `lua/plugins/catppuccin.lua`:
```lua
return {
  "catppuccin",
  opts = function(_, opts)
    -- Color palette: https://github.com/catppuccin/catppuccin?tab=readme-ov-file
    opts.custom_highlights = function(colors)
      return {
        LineNr = { fg = colors.overlay0 },
        CursorLineNr = { fg = colors.crust, bg = colors.yellow },
        WinSeparator = { fg = colors.yellow },
        GitSignsCurrentLineBlame = { fg = colors.overlay0 },
        Visual = { fg = colors.crust, bg = colors.red },
      }
    end
  end,
}
```
